### DISCUSSION TITLE
Where's my Joystick?

### DISCUSSION THREAD

**Discussion by stefanesa** [polymathrobotics/caladan_examples#14] [Q&A] [answered]
Hey all,

I'm in the "Operate" mode of Formant for my sim instance and I can't find my joystick.  Previously when I was testing there was a diamond in the middle of a circle, and when I dragged that diamond I could manually reposition my tractor (as opposed to figuring out commands to get it in a specific position).

How do I get the joystick back?

**Comment by iliabaranov** [comment, marked as answer]
Hello!

In the bottom right corner, select the "unlock" toggle, to show the teleop joystick.
